packages/app/src/cli/models/app/app.ts

```diff
@@ -17,6 +17,7 @@ export enum WebType {
 
 export const WebConfigurationSchema = schema.define.object({
   type: schema.define.enum([WebType.Frontend, WebType.Backend]),
+  authCallbackPath: schema.define.string().default(''),
```

Comment:
I'd use snake case for this one `auth_callback_path` to align with the TOML convention.